libs/ml/examples/commodity_prediction/main.cpp

```diff
@@ -0,0 +1,349 @@
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "file_loader.hpp"
+#include "math/tensor.hpp"
+#include "ml/dataloaders/tensor_dataloader.hpp"
+#include "ml/graph.hpp"
+#include "ml/layers/fully_connected.hpp"
+#include "ml/ops/activation.hpp"
+#include "ml/state_dict.hpp"
+
+#include <fstream>
+#include <iostream>
+#include <ml/ops/transpose.hpp>
```

Comment:
if you're going to separate your includes like this then transpose should be with the top group in ""